src/Event/Android/KeyCode.hpp

```diff
@@ -38,6 +38,20 @@ enum {
   KEY_MENU = 0x52,
   KEY_ESCAPE = 0x6f,
 
+  KEYCODE_BUTTON_R1 = 0x67,
+  KEYCODE_BUTTON_R2 = 0x69,
+  KEYCODE_BUTTON_L1 = 0x66,
+  KEYCODE_BUTTON_L2 = 0x68,
+  KEYCODE_BUTTON_A = 0x60,
+  KEYCODE_BUTTON_B = 0x61,
+  KEYCODE_BUTTON_C = 0x62,
+  KEYCODE_BUTTON_X = 0x63,
+  KEYCODE_BUTTON_Y = 0x64,
+  KEYCODE_BUTTON_Z = 0x65,
+  KEYCODE_MEDIA_NEXT = 0x57,
+  KEYCODE_MEDIA_PREVIOUS = 0x58,
+  KEYCODE_MEDIA_PLAY_PAUSE = 0x55, 
```

Comment:
These key codes should be in a separate commit, with a link to the official documentation where you found their definitions.

Reply:
They are from the same source as the others in the class.